tests/InstrEngineTests/InstrEngineTests/ProfilerHelpers.cs

```diff
@@ -338,7 +339,12 @@ private static void DiffResultToBaselineNode(XmlNode baselineNode, XmlNode outpu
             {
                 if (CompareOrdinalNormalizeLineEndings(baselineNode.Value, outputNode.Value) != 0)
                 {
-                    Assert.Fail("Baseline value does not equal output value\n" + baselineNode.Value + "\n" + outputNode.Value);
+                    int index = baselineNode.Value.Zip(outputNode.Value, (c1, c2) => c1 == c2).TakeWhile(b => b).Count() + 1;
```

Comment:
>(c1, c2) => c1 == c2 [](start = 73, length = 20)

In order to match the check above, should we normalize the endings first?

Reply:
I spent quite a lot of time trying to work out why two nearly identical strings didn't match. Turns out instructions node needs to end with exactly 4 spaces.

Reply:
Yes, we could do a better job with this, for sure.

---
In reply to: [566280346](https://github.com/microsoft/CLRInstrumentationEngine/pull/353#discussion_r566280346) [](ancestors = 566280346)

Reply:
Fixed with: https://github.com/robertpi/CLRInstrumentationEngine/commit/473acd0848dfffc4e41f36ad561a365ea2698a7e